src/wwwroot/controllers/MainCtrl.js

```diff
@@ -63,7 +66,30 @@
         isFreeTrialAlmostEnded : false,
         isFreeTrialEndToday : false
       }
-      
+      console.log(moment($rootScope.getFreeTrialNotificationFromStorage).toDate());
+      console.log( moment(freeTrialEndDate).toDate());
+      console.log(moment($rootScope.getFreeTrialNotificationFromStorage) <= moment(freeTrialEndDate));
+      if (!modalOpened) {        
+        if(!$rootScope.getFreeTrialNotificationFromStorage || moment($rootScope.getFreeTrialNotificationFromStorage).toDate() <= moment(freeTrialEndDate).toDate()) {
```

Comment:
Is it necessary all of this transformation? in that case is it possible to do it in `getFreeTrialNotificationFromStorage` in order to do it less times?